lighthouse-core/runner.js

```diff
@@ -318,9 +318,15 @@ class Runner {
     const computedArtifacts = {};
     const filenamesToSkip = [
       'computed-artifact.js', // the base class which other artifacts inherit
+      'metrics', // the sub folder that contains metric names
     ];
 
-    require('fs').readdirSync(__dirname + '/gather/computed').forEach(function(filename) {
+    const fileList = [
+      ...fs.readdirSync(path.join(__dirname, './gather/computed')),
+      ...fs.readdirSync(path.join(__dirname, './gather/computed/metrics')).map(f => `metrics/${f}`),
```

Comment:
can you check if gulpfile also needs a bump?

Reply:
this effectively is bumping the gulpfile from what I could tell
https://github.com/GoogleChrome/lighthouse/blob/32fd61fee406fe6556c09c382ef82d03f08a84a4/lighthouse-extension/gulpfile.js#L35-L36